src/style.css

```diff
@@ -0,0 +1,82 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  flex-basis: 300px;
+  flex-grow: 1;
+  min-height: 300px;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  color: #fff;
+}
+
+.block--1 {
+  background-color: red;
+}
+
+.block--2 {
+  background-color: rgb(204, 0, 0);
+}
+
+.block--3 {
+  background-color: rgb(153, 0, 0);
+}
+
+.block--4 {
+  background-color: rgb(102, 0, 0);
+}
+
+.block--5 {
+  background-color: rgb(51, 0, 0);
+}
+
+.block--6 {
+  background-color: black;
+}
+
+@media (min-width: 900px) {
+  .block {
+    min-width: 33.33%;
+  }
+  .block--4 {
+    order: 3;
+  }
+  .block--5 {
+    order: 2;
+  }
+  .block--6 {
+    order: 1;
+  }
+}
+
+@media (min-width: 600px) and (max-width: 899px) {
```

Comment:
```suggestion
@media (min-width: 600px) {
```
Do you really need a second one? Now it works like this
![media](https://user-images.githubusercontent.com/96409563/225259227-8bca948e-4af4-4658-bc1f-85d8773f9c0b.gif)